### ISSUE TITLE
Bug in InsertBlankLineBetweenFuncProcCheck

### ISSUE THREAD

**Issue by ThomasPohl** [EXXETA/sonar-esql-plugin#31]
The InsertBlankLineBetweenFuncProcCheck seems to produce false positive issues. The example below produces three issues.
I would fix this bug, but I'm not really sure what the intended behavior  is.

```
CREATE COMPUTE MODULE Module1

	CREATE FUNCTION Main() RETURNS BOOLEAN
	BEGIN
		DECLARE myInt INTEGER 1;
		DECLARE myInt2 INTEGER;
	END;
		
	CREATE Procedure Main2() RETURNS BOOLEAN
	BEGIN
		DECLARE myInt INTEGER 1;
		BEGIN
			DECLARE myInt2 INTEGER;
		END;
	END;
	
END MODULE;
```

**Comment by sapna392** [comment]
Bug fixed for InsertBlankLineBetweenFuncProcCheck , i have checked in and created pull request , please check and let me know in case of any concerns.

**Comment by ThomasPohl** [comment]
Thank you for the fix. I have merged your changes. Somehow github doesn't recognize the merge as a merge but marked the pull request as closed...
When I find some time I will update the rule to use the parsed tree (AST) to determine missing blank lines.
